modules/tensor_mechanics/doc/content/source/materials/ComputeLinearElasticPFFractureStress.md

```diff
@@ -229,6 +229,11 @@ Integration by parts yields
 The boundary integral term can be neglected as in most applications $d=0$ on $\partial\Omega$. Some authors [!cite](Mikelic2019) have proposed to replace the indicator function $d$ with $d^2$ in the first term in order to make the functional convex. The indicator function is implemented as a generic material object that can be easily provided and modified in an input file.
 The stress equilibrium and damage evolution equations are also modified to account for the pressure contribution.
 
+## PETSc SNES variational inequalities solver option
+
+Alternatively, the damage irreversibility condition can be enforced by using PETSc's SNES variational inequalities (VI) solver. In order to use PETSc's VI solver, [`PFFractureBoundsAux`](/phase_field/PFFractureBoundsAux.md) is required to set the bounds of phase field fracture variable. Note that in order for these bounds to have an effect, the user has to specify the
```

Comment:
@sapitts

Reply:
@jiangwen84 it looks like it is not possible to link from a phase field md file in a tensor mechanics file like you are trying to do here. While we do have some examples in the tensor mechanics documentation of linking to a phase field test, e.g.
```
moose/modules/tensor_mechanics/doc/content/source/materials/ComputeCrackedStress.md:!listing modules/combined/test/tests/phase_field_fracture/crack2d_computeCrackedStress_smallstrain.i
```
We don't have ability to link to a phase field md file, e.g.
```
moose/modules/tensor_mechanics/doc/content/modules/tensor_mechanics/index.md:and in combined physics simulations with the Heat Transfer, Phase Field, Contact, ....
```
I believe this limitation is by design to speed up the building of the MooseDocs, or is a function of how the MooseDocs uses the executable from each module.  In any case, the best option is to remove the link for the the `PFFractureBoundsAux` class.

Reply:
Thanks!